app/controllers/user.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const bcrypt = require('bcrypt'),
+  saltRounds = 10,
+  salt = bcrypt.genSaltSync(saltRounds);
+
+const User = require('../models').users;
+
+const validateEmail = valor => {
+  if (/^\w+([\.-]?\w+)@wolox+(\.\w{2,3})+$/.test(valor)) {
+    return true;
+  } else {
+    return false;
+  }
+};
+
+const validatePassword = valor => {
+  if (/^(?=\w*\d)(?=\w*[A-Z])(?=\w*[a-z])\S{8,}$/.test(valor)) {
+    return true;
+  } else {
+    return false;
+  }
+};
+
+const singUp = (req, res) => {
+  const user = {};
+
+  user.firstName = req.body.firstName ? req.body.firstName : '';
```

Comment:
This assignations are unnecessary, replace with something like this `const user = req.body ? { firstName: req.body.firstName .... } : {}` if any are undefined nothing is set to `firstName`